Shared/Samples/Monitor changes to map load status/MonitorChangesToMapLoadStatusView.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct MonitorChangesToMapLoadStatusView: View {
+    /// A map with an imagery base map.
+    @State private var map = Map(basemapStyle: .arcGISImagery)
+    
+    /// A string indicating the load status of the map.
+    @State private var loadStatusText = LoadStatus.notLoaded.title
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: map)
+            .overlay(alignment: .top) {
+                Text("Load Status: \(loadStatusText)")
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .task {
+                // Listen for load status changes.
+                for await loadStatus in map.$loadStatus {
+                    // Set the load status text when the load status updates.
```

Comment:
```suggestion
```
This documentation is not necessary.

Reply:
Hmm that is a good question. @yo1995 What is the line for documentation in Samples? I understand adding some doc that might seem unnecessary in certain areas could be helpful for a newcomer but when should we say a piece of documentation won't be helpful for anyone?

Reply:
Depending on the audience for a sample. I'd say if a sample is more entry level like this one (or Display map, Show callout, etc.), adding more comments is OK. Samples like utility network and offline workflows, which target advanced users, don't need these trivial comments. They only need to focus on the key logics.